doc/march_packages/march_moveit.rst

```diff
@@ -0,0 +1,37 @@
+.. _march-moveit-label:
+
+march_moveit
+============
+
+Overview
+--------
+The march_moveit package is the package that contains all files necessary for the integration of
+`MoveIt <https://moveit.ros.org/>`_. MoveIt is an open source project that is the result of the combined efforts
+of a large international community and multiple organizations. MoveIt is a framework for everything that has to do
+with motion planning. There are a lot of `tutorials <https://ros-planning.github.io/moveit_tutorials/>`_ on the project.
+
+We started using MoveIt for implementing capture point. More information on that is in :ref:`march-gait-selection-label`.
+Here we use the `Python MoveIt Commander <http://docs.ros.org/jade/api/moveit_commander/html/index.html>`_ as the interface to MoveIt.
+The integration allows to use MoveIt for all kinds of dynamic gaits.
+
+The files in this package are generated by the MoveIt Setup Assistant. Manually editing the files in this packages is not
+recommended.
+
+ROS API
+-------
+
+The packages has a very extensive ROS API, because MoveIt is very well integrated with ROS. This will not all be listed
+here as most of it is only used by MoveIt itself. An important thing to note is that MoveIt can publish position and
+velocity commands directly to the controller. We are avoiding this, because we want to stay in control.
+
+
+Tutorials
+---------
+
+How to use the setup-assistant
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    roslaunch moveit_setup_assistant setup_assistant.launch
+
+Select the `march_moveit` as the package to alter.
+You can now change the MoveIt integration. The necessary files will be automatically generated.
```

Comment:
Maybe add a link to MovIt documentation here?

Reply:
There are three links to MoveIt docs above, is this really necessary?

Reply:
Mwah you could add a direct link to https://ros-planning.github.io/moveit_tutorials/doc/setup_assistant/setup_assistant_tutorial.html but not essential I guess

Reply:
Mwah you could add a direct link to https://ros-planning.github.io/moveit_tutorials/doc/setup_assistant/setup_assistant_tutorial.html but not essential I guess

Reply:
See this

Reply:
@RutgerVanBeek

Reply:
you are right, will do